### PR TITLE
except ConnectionResetError in MultiProcessingHandler

### PR DESCRIPTION
**Pull request by jstriebel**
### Description:
Before, I got the following stacktrace at the end of successful tests:
```
Traceback (most recent call last):
  File "/…/cluster_tools/cluster_tools/multiprocessing_logging_handler.py", line 50, in _receive
    record = self.queue.get(timeout=0.2)
  File "<string>", line 2, in get
  File "/usr/lib/python3.7/multiprocessing/managers.py", line 796, in _callmethod
    kind, result = conn.recv()
  File "/usr/lib/python3.7/multiprocessing/connection.py", line 250, in recv
    buf = self._recv_bytes()
  File "/usr/lib/python3.7/multiprocessing/connection.py", line 407, in _recv_bytes
    buf = self._recv(4)
  File "/usr/lib/python3.7/multiprocessing/connection.py", line 379, in _recv
    chunk = read(handle, remaining)
ConnectionResetError: [Errno 104] Connection reset by peer
```

This is now handled properly. ~~I also suspect that this error might be the reason for the current CI failures, let's see.~~

### Todos:
 - [x] Updated Changelog

### CODE REVIEW

**Comment by jstriebel** [comment]
Even though this does not resolve the CI issues, I believe this is still useful.

File: cluster_tools/Changelog.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ For upgrade instructions, please check the respective *Breaking Changes* section
 ### Changed
 
 ### Fixed
+- Suppress spurios ConnectionResetErrors when using multiprocessing, probably only relevant for Python 3.9.13 and higher. [#770](https://github.com/scalableminds/webknossos-libs/pull/770)
```
Comment (philippotto) [inline cluster_tools/Changelog.md:19]:
```suggestion
- Suppress spurious ConnectionResetErrors when using multiprocessing, probably only relevant for Python 3.9.13 and higher. [#770](https://github.com/scalableminds/webknossos-libs/pull/770)
```

File: cluster_tools/cluster_tools/multiprocessing_logging_handler.py [hunk 2]
```diff
@@ -54,7 +54,12 @@ def _receive(self) -> None:
                 raise
             # multiprocessing.managers.RemoteError pop up quite often.
             # It seems that they can be safely ignored, though.
-            except (BrokenPipeError, EOFError, multiprocessing.managers.RemoteError):
+            except (
```
Comment (philippotto) [inline cluster_tools/cluster_tools/multiprocessing_logging_handler.py:57]: the comment in lines 55f could probably be adapted a bit, right?

**Review by philippotto — APPROVED** [review]
only got two nitpicks :)

**Pull request merged**